### PR TITLE
Add a Shipping View for PHSKC Encounter Metadata

### PR DESCRIPTION
**Pull request by bencap**
This change adds a shipping view for PHSKC encounter metadata and the
`result_value` of external UW Virology PA results associated with
each encounter.

### CODE REVIEW

**Review by davereinhart — COMMENTED** [review]
Rather than using select subqueries for each column, you might consider using filters (should perform better with less code). See this commit where a similar view was refactored:
https://github.com/seattleflu/id3c-customizations/commit/333b555895bc0674415fdf7a74ac23e61c128c2f

**Comment by bencap** [comment]
@davereinhart Thanks for the suggestion, it is definitely a lot cleaner that way.

**Comment by bencap** [comment]
[PHSKC - Develop shipping views for Amanda/Chelsea - Part 3](https://trello.com/c/Ac5QVyqL/1283-phskc-develop-shipping-views-for-amanda-chelsea-part-3)

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -1013,6 +1014,74 @@ grant select
...
+            from warehouse.encounter
+              left join warehouse.sample using (encounter_id)
+              left join warehouse.presence_absence as pa using (sample_id)
+            where target_id = 952
+            group by encounter_id, present
+        )
+
+    select
+        encounter_id,
+        extract(years from age) as age,
```
Comment (davereinhart) [inline schema/deploy/shipping/views.sql:1063]: Could use the `age_in_years(interval)` function here for consistency with other shipping views.

**Comment by bencap** [comment]
Deployed via sqitch

**Pull request merged**